### DISCUSSION TITLE
Is it possible to run in android? [ROOT]

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [vfsfitvnm/frida-il2cpp-bridge#455] [Q&A]
Termux is giving many errors even I placed all files it in termux data directory

If anyone have tried in past, please help
![Screenshot_20231230-010543](https://github.com/vfsfitvnm/frida-il2cpp-bridge/assets/49313374/0678ce24-1dba-46b7-b018-20aa9f0f79c6)

**Comment by vfsfitvnm** [comment]
It looks like you need to install https://www.npmjs.com/package/@types/frida-gum

**Reply by RevealedSoulEven** [reply]
1:34:12 AM - Found 0 errors. Watching for file changes.




The script is still not terminated. What does this mean?

**Reply by RevealedSoulEven** [reply]
Ahhh anyways. It's working. Thanks a lot 😊
